woocommerce/woocommerce-ios
# Adjust TitleAndTextFieldTableViewCell to use a fixed width

Summary
==========
Fix issue #5084 by setting a defined width to the Address field labels in order to keep the field content with the same vertical alignment. I'm not sure if this would be the best approach considering the variation of text sizes from different languages, but I decide to follow this inspired by how it was done in the Order Creation Address form.

Screenshots
==========
| Before  | After |
| ------------- | ------------- |
| ![IMG_8878](https://user-images.githubusercontent.com/5920403/165644891-4f6a20c4-86ee-421c-ad68-0b07297d22fc.PNG) | ![IMG_8877](https://user-images.githubusercontent.com/5920403/165644912-17ca2fe0-bbfa-4222-a904-2bdd31d0ab1f.PNG) |

How to Test
==========
1. Make sure your Woo site supports the Shipping Label plugin and it's correctly configured to work with it
2. Select an Order with the status as `Processing` and select `Create Shipping Label`
3. At the `Create Shipping Label` view, select the `Continue` button under the `Ship From` Address info
4. Verify that all Address fields are correctly aligned as expected.

Update release notes:
